src/utils/configuration.ts

```diff
@@ -90,6 +91,10 @@ const configurationSchema = {
     maskOutputs: z.boolean().default(true),
     loginPrompt: z.boolean().default(true),
     platformAuth: z.boolean().default(false),
+    docsUrl: z.string().default(DEFAULT_DOCS_URL),
+  },
+  beta: {
+    shellWarning: z.boolean().default(true),
```

Comment:
@pastuxso default `true`? again why not use experiments instead of `beta`

Reply:
🙌  Well spotted! It's crucial to ensure that I'm testing beyond just the happy path.